src/commands/workflow/run.ts

```diff
@@ -0,0 +1,48 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { commaSeparatedStr, jsonStr } from "@/lib/helpers/flag";
+import { withSpinner } from "@/lib/helpers/request";
+import { indentString } from "@/lib/helpers/string";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
+    environment: Flags.string({
+      default: "development",
+      summary:
+        "The environment in which to run the workflow; defaults to development.",
+    }),
+    recipients: commaSeparatedStr({
+      required: true,
+      aliases: ["recipient"],
+      summary:
+        "A comma-separated list of user IDs that will receive this workflow. Max number of recipients is 5.",
```

Comment:
```suggestion
        "One or more recipient ids for this workflow run, separated by comma.",
```
A couple edits for your considerations:
* Removed the "user ids" bit, because I don't think recipients have to be users only. (i.e. objects)
* I removed the bit about the max recipients count because that validation logic only lives in the backend. So if that number ever gets updated in the backend, then the --help doc would be wrong and it will require remember to cut a new release. If they ever passed in more than 5, then they would receive/see the error message given by the backend.